app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt

```diff
@@ -285,7 +285,7 @@ class PatientEntryScreen(context: Context, attrs: AttributeSet) : RelativeLayout
   }
 
   override fun setColonyOrVillagesAutoComplete(colonyOrVillageList: List<String>) {
-    ArrayAdapter<String>(context, R.layout.village_typeahead_list_item, R.id.villageTypeAheadItemTextView, colonyOrVillageList).also { adapter ->
+    VillageTypeAheadAdapter(context, colonyOrVillageList).also { adapter ->
```

Comment:
Let's follow one approach, in the earlier comment we used a variable name for the adapter and set to the text field. Let's stick with that.
```kt
    val adapter = VillageTypeAheadAdapter(context, colonyOrVillageList)
    colonyOrVillageEditText.setAdapter(adapter)
```